geniza/corpus/templates/corpus/snippets/document_transcription.html

```diff
@@ -16,36 +16,35 @@
     {% endif %}
 {% endif %}
 {# only enable ITT panel if transcription or images are present, or we are in edit mode #}
-{% if document.has_transcription or document.iiif_urls or edit_mode %}
+{% if document.has_transcription or document.has_translation or document.iiif_urls or edit_mode %}
```

Comment:
worth consolidating these checks into a single method on the document? what's the generic for all of these, has content? has text or images?

Reply:
I think that's a good idea. And I just noticed that `has_translation` is still only checking for print translations. I wonder (for both admin and public site) if the researchers want that to change to only digitized translations. I will ask!